<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlbumWidget</class>
 <widget class="QWidget" name="AlbumWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="albumInfoFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="albumName">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Lorem Ipsum</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addPicturesButton">
        <property name="text">
         <string>Add Pictures</string>
        </property>
        <property name="icon">
         <iconset resource="res/resource.qrc">
          <normaloff>:/icons/photo-add.png</normaloff>:/icons/photo-add.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editButton">
        <property name="text">
         <string>Edit</string>
        </property>
        <property name="icon">
         <iconset resource="res/resource.qrc">
          <normaloff>:/icons/album-edit.png</normaloff>:/icons/album-edit.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="text">
         <string>Delete</string>
        </property>
        <property name="icon">
         <iconset resource="res/resource.qrc">
          <normaloff>:/icons/album-delete.png</normaloff>:/icons/album-delete.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="thumbnailListView"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res/resource.qrc"/>
 </resources>
 <connections/>
</ui>
